index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Best Grill</title>
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+      <link rel="stylesheet" href="main.css">
+  </head>
+       <body>
+
+        <div class="top-links-container">
+          <div class="top-links flex">
+              <a href="">
+                  <h4>English</h4>
+              </a>
+              <a href="">
+                  <h4>My Page</h4>
+              </a>
+              <a href="">
+                  <h4>Logout</h4>
+              </a>
+              <a href="#twitter"><i class="fa fa-twitter"></i></a>
+              <a href="#facebook"><i class="fa fa-facebook-f"></i></a>
+             
+          </div>
+      </div>
+             <header>
+                    <div class="container">
+                        <div>
+                          <h2 class="logo"><a href="index.html"><img src="./components/resto.png" alt=""></a></h2>
+                          <div class="nav-toggle" id="navToggle">
+                             <img id="hamburger" class="navIcon" src="./components/list.svg" alt="hamburger">
+                             <img class="navIcon hidden" src="./components/cross.svg" alt="close">
+                          </div>
+                        </div>
+                        <nav>
+                          <ul>
+                            <li class="red"><a href="./assets/about.html">About</a></li>
+                            <li class="disable"><a href="#">Services</a></li>
+                            <li class="disable"><a href="#">Gallery</a></li>
+                            <li class="disable"><a href="#">Pricing</a></li>
+                            <li class="disable"><a href="#">Contact us</a></li>
+                            <li class="campain"><a href="">CC Campain</a></li>
+                          </ul>
+                        </nav>
+                    </div> 
+                    <div>
+                      
+                    </div>
+                    <div class="main">
+                      <h2>"Hello Sharing World"</h2>
+                      <h1>WELCOM TO THE BEST GRILL</h1>
```

Comment:
- [ ] OPTIONAL: There is a typo here!  "WELCOM" should be "WELCOME".  Please, I recommend you fix this.

Reply:
Hi @Meltrust, thank you for approving my PR and mentioning this type, I fixed it